hp_list/Makefile

```diff
@@ -12,6 +12,17 @@ $(BIN): main.c
 all: CFLAGS += -O2
 all: $(BIN)
 
+# The RUNTIME_STAT allow to show the runtime operation states
```

Comment:
Shorten the descriptions. You conceptually mention "runtime statistics of atomic operations."